backend/app/routers/files.py

```diff
@@ -409,3 +410,43 @@ async def resubmit_file_extractions(
             FileOut(**file.dict()), rabbitmq_client, user, credentials
         )
     return resubmit_success_fail
+
+@router.post("/{file_id}/visualization")
+async def save_visualization(
```

Comment:
can you rename this to visualization_config, may be? I named visualization in my PR for the data

Reply:
Everything should be renamed now. 

These will all be in separate routes in visualizations router.

Reply:
I see the separate router. looks good. Do you still need these in files.py?

Reply:
No, I will remove them.